presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -106,8 +106,7 @@
     public void testDuplicateRelation()
             throws Exception
     {
-        assertFails(DUPLICATE_RELATION, "SELECT * FROM t1 JOIN t1 USING (a)");
-        assertFails(DUPLICATE_RELATION, "SELECT * FROM t1 x JOIN t2 x USING (a)");
+        assertFails(DUPLICATE_RELATION, "WITH x as (SELECT 1), x as (SELECT 2) SELECT * from x");
```

Comment:
This test seems unrelated. 

Also, I'm not even sure that's invalid, but I need to check the SQL spec. It's possible that the second `x` hides the first one.